.github/workflows/test.yml

```diff
@@ -271,7 +271,7 @@ jobs:
         uses: codecov/codecov-action@v4
 
   test-unix-pandas1:
-    needs: test-nosoftdeps
+    needs: [test-base, test-module, test-other]
```

Comment:
The above doubt remains, but may be here we can use test-full to reduce the number of simultaneous jobs a bit. Haven't thought in details, just a spontaneous suggestion.